.flake8

```diff
@@ -0,0 +1,11 @@
+[flake8]
+doctests = True
+exclude =
+    **/__init__.py
+    **/tests/*
+    *build/
+    docs/sphinxext/
+    docs/tools/
+    reproschema/_version.py
+max-line-length=79
+extend-ignore = B001, B006, B016, E501, E722, F821
```

Comment:
I can fix those errors but probably in a follow up PR.

Reply:
maybe another PR later